examples/apps/inside_out_demo/workload/inside_out_eval_logic.cpp

```diff
@@ -0,0 +1,109 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include <string.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <vector>
+
+#include "inside_out_eval_logic.h"
+#include "trusted_iohandler.h"
+#include "enclave_utils.h"
+
+// Maximum size of buffer that is used for file io
+size_t GetMaxFileSize() {
+    return 1024;
+}
+
+// Maximum size of result buffer that is used to store status of io
+size_t GetMaxIoResultSize() {
+    return 128;
+}
+
+std::string ReadFile(uint32_t handlerId, std::string fileName) {
+    std::string outBuf;
+    std::string result;
+    size_t resultSize = GetMaxIoResultSize();
+    size_t outBufSize = GetMaxFileSize();
+    outBuf.reserve(outBufSize);
+    result.reserve(resultSize);
+    uint32_t status = FileRead(handlerId, fileName, (uint8_t *)result.c_str(),
+        resultSize, (uint8_t *)outBuf.c_str(), outBufSize);
+
+    if (status == 0) {
+        Log(TCF_LOG_INFO, "File Read operation success. File content: %s",
+            outBuf.c_str());
+    }
+    else {
+	return result;
```

Comment:
May be make sense to add a comment where and how result and outBuf gets filled

Reply:
Done